client/i18n/locales/chinese/translations.json

```diff
@@ -1,444 +1,444 @@
+      "confirm": "确认新邮箱",
+      "weekly": "给我发 Quincy 的每周订阅邮件"
     },
     "honesty": {
-      "p1": "Before you can claim a verified certification, you must accept our Academic Honesty Pledge, which reads:",
-      "p2": "\"I understand that plagiarism means copying someone else’s work and presenting the work as if it were my own, without clearly attributing the original author.\"",
-      "p3": "\"I understand that plagiarism is an act of intellectual dishonesty, and that people usually get kicked out of university or fired from their jobs if they get caught plagiarizing.\"",
-      "p4": "\"Aside from using open source libraries such as jQuery and Bootstrap, and short snippets of code which are clearly attributed to their original author, 100% of the code in my projects was written by me, or along with another person going through the freeCodeCamp curriculum with whom I was pair programming in real time.\"",
-      "p5": "\"I pledge that I did not plagiarize any of my freeCodeCamp.org work. I understand that freeCodeCamp.org’s team will audit my projects to confirm this.\"",
-      "p6": "In the situations where we discover instances of unambiguous plagiarism, we will replace the person in question’s certification with a message that \"Upon review, this account has been flagged for academic dishonesty.\"",
-      "p7": "As an academic institution that grants achievement-based certifications, we take academic honesty very seriously. If you have any questions about this policy, or suspect that someone has violated it, you can email <0>{{email}}</0> and we will investigate."
+      "p1": "申请认证前，你必须先接受我们的《学术诚信条例》，即：",
+      "p2": "\"我理解剽窃意味着抄袭他人的作品，将其作为自己的作品来提交，且不明确注明原作者。\"",
+      "p3": "\"我理解剽窃是一种知识上的不诚实行为，如果有人被发现有这种行为，那么他们通常会被大学（或其他单位）解雇。\"",
+      "p4": "\"除了使用开源代码库（如 jQuery，Bootstrap）以及明确注明原作者的的代码片段之外，我的项目中的代码 100％ 是由我自己编写的，或者是与另一位 freeCodeCamp 学员结对编程时编写的。\"",
+      "p5": "\"我保证我提交给 freeCodeCamp 的作品都不是剽窃他人的。我理解 freeCodeCamp 团队将审核我提交的项目，以确认这一点。\"",
+      "p6": "如果我们发现某学员的某部分课程项目有明显的剽窃现象，我们将不会发布该项目的认证，并注明 \"经过审核，该帐户已被标记为违反学术诚信条例。\"",
+      "p7": "作为一个基于成就发布认证的学术机构，我们非常重视学术诚信。如果你对此条例有任何疑问，或者怀疑有人违反了该条例，可以发送邮件至 <0>{{email}}</0>，我们将进行调查。"
     }
   },
   "profile": {
-    "you-not-public": "You have not made your portfolio public.",
-    "username-not-public": "{{username}} has not made their portfolio public.",
-    "you-change-privacy": "You need to change your privacy setting in order for your portfolio to be seen by others. This is a preview of how your portfolio will look when made public.",
-    "username-change-privacy": "{{username}} needs to change their privacy setting in order for you to view their portfolio.",
+    "you-not-public": "你尚未把你的作品集设置为公开。",
+    "username-not-public": "{{username}} 未把作品集设置为公开",
+    "you-change-privacy": "你需要更改你的隐私设置，才能让他人看到你的作品集。这是你的作品集公开时候的预览。",
+    "username-change-privacy": "{{username}} 需要修改隐私设置，你才能看到其作品集。",
     "supporter": "Supporter",
     "contributor": "Top Contributor",
```

Comment:
No translations or something for these two I take it?

Reply:
I suggest we just keep the special titles as English, for consistency with the global community.

Reply:
I think we should leave the titles as is.